ui/partials/perf/graphsctrl.html

```diff
@@ -19,6 +19,8 @@
                   {{series.projectName}}<br/>
                   {{series.platform}}<br/>
                   <div class="signature"><small>{{series.signature}}</small></div>
+                  <div><a ng-click="addRelatedPlatforms(series.signature)">Add all related platforms</a></div>
```

Comment:
I don't think we want to add these elements to the UI -- it's going to make things much busier for something that will only be useful a small proportion of the time. Instead, why not linkify the series.projectName and series.platform above? That goes along with what we do elsewhere in the treeherder UI.